corehq/apps/accounting/models.py

```diff
@@ -2874,12 +2876,13 @@ def generate_pdf(self, invoice):
 
         template.get_pdf()
         filename = self.get_filename(invoice)
+        blob_domain = domain or "<unknown>"
```

Comment:
nit: how about extracting `<unknown>` as a const like, `UNKNOWN_DOMAIN_PLACEHOLDER` or something similar instead of keeping a string here which can possibly be used again or easy to change later at a single spot if needed?

Reply:
And is this expected? to have an invoice without domain?